website/docs/components/data-connectors/github.md

```diff
@@ -227,31 +226,40 @@ datasets:
-| closed_at      | Timestamp  | YES         |
-| comments_count | Int64      | YES         |
-| commits_count  | Int64      | YES         |
-| created_at     | Timestamp  | YES         |
-| deletions      | Int64      | YES         |
-| hashes         | List(Utf8) | YES         |
-| id             | Utf8       | YES         |
-| labels         | List(Utf8) | YES         |
-| merged_at      | Timestamp  | YES         |
-| number         | Int64      | YES         |
-| reviews_count  | Int64      | YES         |
-| state          | Utf8       | YES         |
-| title          | Utf8       | YES         |
-| url            | Utf8       | YES         |
+| Column Name     | Data Type                                                       | Is Nullable |
+| --------------- | --------------------------------------------------------------- | ----------- |
+| additions       | Int64                                                           | YES         |
+| assignees       | List(Utf8)                                                      | YES         |
+| author          | Utf8                                                            | YES         |
+| body            | Utf8                                                            | YES         |
+| changed_files   | Int64                                                           | YES         |
+| closed_at       | Timestamp                                                       | YES         |
+| comments_count  | Int64                                                           | YES         |
+| commits_count   | Int64                                                           | YES         |
+| created_at      | Timestamp                                                       | YES         |
+| deletions       | Int64                                                           | YES         |
+| hashes          | List(Utf8)                                                      | YES         |
+| id              | Utf8                                                            | YES         |
+| labels          | List(Utf8)                                                      | YES         |
+| merged_at       | Timestamp                                                       | YES         |
+| number          | Int64                                                           | YES         |
+| reviews_count   | Int64                                                           | YES         |
+| state           | Utf8                                                            | YES         |
+| title           | Utf8                                                            | YES         |
+| url             | Utf8                                                            | YES         |
+| discussion      | List(Struct(body: Utf8, author: Utf8, created_at: Timestamp))   | YES         |
+| review_comments | List(Struct(body: Utf8, author: Utf8, created_at: Timestamp))   | YES         | 
```

Comment:
Consider adding to list in alphabetical order like the others. It just makes scanning the list for a particular column name quicker for the reader.